src/aiortc/codecs/h264.py

```diff
@@ -277,6 +277,9 @@ def _encode_frame(
             self.buffer_pts = None
             self.codec = None
 
+        # reset the picture type, otherwise no B-frames are produced
+        frame.pict_type = av.video.frame.PictureType.NONE
```

Comment:
Why necessary?

Reply:
See the discussion in the original ticket.